### DISCUSSION TITLE
Generate Keystone schema from Prisma

### DISCUSSION THREAD

**Discussion by devmanbr** [keystonejs/keystone#8296] [Feature requests]
I'm really new to Keystone. I have been a Prisma user for some time now.
I did a lot of research about how to get started, and all that.

apparently, according to what I could find, there is no way to automatically "convert" Prisma schemas to Keystone, which ends up requiring manual work (significant, if we consider schemas that have 100 to 120 models).

are there any plans to implement something to facilitate this aspect?

I opened the discussion because this is important to me. I left adminjs because the experience was not very pleasant in development.
since Keystone uses Prisma and Next.js, which is my current base stack, would be a good choice and starting point, but only if it doesn't mean too much work for every new project

**Comment by JaLe29** [comment]
Same there, I would like to generate keystone app based on my prisma schema.  Not prisma schema from keystone configuration... :-)

Something like this https://github.com/brookmg/prisma2keystone

**Reply by devmanbr** [reply]
Seeing this answer only today. sorry.

the solution seems interesting, but does it really manage to cover everything that the prism offers in the scheme?
